csfieldguide/templates/interactives/matrix-simplifier.html

```diff
@@ -0,0 +1,190 @@
+
+{% load i18n %}
+{% load static %}
+
+{% block html %}
+  <div class="container">
+    <h3 class='text-center'>{% trans "Matrix Simplifier" %}</h3>
+    <p class="text-center">{% trans "Try dragging the matrices and vectors below to change their order. Does this affect the output?" %}</p>
+    <div class="row">
+      <div class="col-md-6 col-sm-12 text-center mb-3">
+        <h5>{% trans "Matrices" %}</h5>
+        <div id="matrix-input-container" class="containers">
+          <div class="draggable content border rounded m-1 center-block">
+            <button id="close-matrix-0" type="button" class="close dismiss-eqtn" aria-label="Close">
+              <span aria-hidden="true">&times;</span>
+            </button>
+            <div id="matrix-0" class="matrix invisible" data-matrix-order=0>
+              \[
+                \begin{bmatrix}
+                cos(45)   &  0  &  sin(45) \\
+                0         &  1  &  0 \\
+                -sin(45)  &  0  &  cos(45) \\
+                \end{bmatrix}
+              \]
+            </div>
+          </div>
+      
+          <div class="draggable content border rounded m-1 center-block">
+            <button id="close-matrix-1" type="button" class="close dismiss-eqtn" aria-label="Close">
+              <span aria-hidden="true">&times;</span>
+            </button>
+            <div id="matrix-1" class="matrix invisible" data-matrix-order=1>
+              \[
+                \begin{bmatrix}
+                10  &  0  &  0 \\
+                0   &  10 &  0 \\
+                0   &  0  &  10 \\
+                \end{bmatrix}
+              \]
+            </div>
+          </div>
+        </div>
+        <button id="add-matrix-btn" type="button" class="btn btn-primary d-none" data-toggle="modal" data-target="#matrix-modal">{% trans "Add matrix" %}</button>
+      </div>
+
+      <div class="col-md-6 col-sm-12 text-center mb-3">
+        <h5>{% trans "Vectors" %}</h5>
+        <div id="vector-input-container" class="containers">
+          <div class="draggable content border rounded m-1 center-block">
+            <button id="close-vector-0" type="button" class="close dismiss-eqtn" aria-label="Close">
+              <span aria-hidden="true">&times;</span>
+            </button>
+            <div id="vector-0" class="vector invisible" data-vector-order=0>
+              \[
+                \begin{bmatrix}
+                10 \\
+                0 \\
+                0 \\
+                \end{bmatrix}
+              \]
+            </div>
+          </div>
+        </div>
+        <button id="add-vector-btn" type="button" class="btn btn-primary d-none" data-toggle="modal" data-target="#vector-modal">{% trans "Add vector" %}</button>
+      </div>
+    </div>
+
+    <div class="row">
+      <h5 class="col-12 text-center">{% trans "Result" %}</h5>
+    </div>
+
+    <div class="row">
+      <div id="output-container" class="col-12 text-center invisible">
```

Comment:
> Could these stay centered on the page but be left-aligned respective to each other?

Yeah they can, the only thing to note is the difference on _some_ mobile phones:

left-aligned:
![image](https://user-images.githubusercontent.com/16066822/64082794-f576b080-cd68-11e9-9fd8-26ad9c0b0638.png)


centered:
![image](https://user-images.githubusercontent.com/16066822/64082796-058e9000-cd69-11e9-86f4-69d6f086b6fa.png)

Aside from this instance, I prefer left aligned as generally we should have the equal signs in line. Just bringing this up in case the mobile format changes things.

Reply:
Could you force the lines to scroll rather than wrap?